console/console-init/ui/src/modules/device/components/CreateDeviceButton/CreateDeviceButton.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React, { useState } from "react";
+import { Dropdown, DropdownToggle, DropdownItem } from "@patternfly/react-core";
+import { ThIcon, CaretDownIcon } from "@patternfly/react-icons";
+
+export interface ICreateDeviceProps {
+  inputInfo: () => void;
+  inputJSON: () => void;
+}
+
+export const CreateDeviceButton: React.FunctionComponent<ICreateDeviceProps> = (
+  inputInfo,
+  inputJSON
+) => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const dropdownItems = [
```

Comment:
we have already one custom dropdown component DropdownWithToggle. Can you try to use that.

Reply:
I am trying to do it, but it seems the `DropdownWithToggle` will need some slight modifications to be able to do that.

Reply:
Sticking to `CreateDeviceButton`.